taipy/core/_scheduler/_job_dispatcher.py

```diff
@@ -53,14 +53,14 @@ def _dispatch(self, job: Job):
         else:
             job.skipped()
             _JobManager._set(job)
-            self.__unlock_edition_on_outputs(job)
+            self.__unlock_edit_on_outputs(job)
             self.__logger.info(f"job {job.id} is skipped.")
 
     @staticmethod
     def _needs_to_run(task: Task) -> bool:
         """
-        Returns True if the task outputs are in cache and if the output's last edition date is prior the input's last
-        edition date.
+        Returns True if the task outputs are in cache and if the output's last edit date is prior the input's last
+        edit date.
```

Comment:
I find this commentary really complex. Can we have something simpler ?

For ex:
```suggestion
        Returns True if the task has no output or if at least one input was modified since the latest run.
```

Reply:
better phrasing, indeed.